### PR TITLE
Remove `MetaClass`

### PR DESCRIPTION
**Pull request by sajith**
Addresses #28.

### CODE REVIEW

**Comment by congwang09** [comment]
Just for future reference, the `swagger_server/messaging/message_queue_producer.py` is not currently being used, we are currently using rpc_queue and topic_queue for communication between SDX LC and SDX controller, but it would be good to keep the message_queue, maybe for future BAPM implementation.

**Review by congwang09 — APPROVED** [review]

File: swagger_server/messaging/message_queue_producer.py [hunk 1]
```diff
@@ -3,20 +3,11 @@
...
-            return cls._instance[cls]
-
-
-class RabbitmqConfigure(metaclass=MetaClass):
+class RabbitMqParams:
     def __init__(
         self,
         queue="hello",
-        host="aw-sdx-monitor.renci.org",
+        host="localhost",
```
[the remark this answers is not in the document]
Comment (sajith) [reply swagger_server/messaging/message_queue_producer.py:10]: Do you think we should also add a port parameter, or is it safe to assume that message queue will be always on the default port?

[the remark this answers is not in the document]
Comment (congwang09) [reply swagger_server/messaging/message_queue_producer.py:10]: I think we should use the port environment variable, there's one in docker-compose `MQ_PORT`

[the remark this answers is not in the document]
Comment (sajith) [reply swagger_server/messaging/message_queue_producer.py:10]: Filed issue #49. Let's continue this discussion there?

**Pull request merged**